graphql/query_resolver.go

```diff
@@ -1024,6 +1024,100 @@ func (r *queryResolver) Version(ctx context.Context, versionID string) (*restMod
 	return &apiVersion, nil
 }
 
+// Image is the resolver for the image field returning information about an image including kernel, version, ami, name, and last deployed time.
+func (r *queryResolver) Image(ctx context.Context, imageID string) (*Image, error) {
+	config, err := evergreen.GetConfig(ctx)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting evergreen configuration: %s", err.Error()))
+	}
+	c := thirdparty.NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	optsHistory := thirdparty.DistroHistoryFilterOptions{
+		Distro: imageID,
+	}
+	resultHistory, err := c.GetHistory(ctx, optsHistory)
```

Comment:
I would maybe suggest moving all of this logic to the runtime_environments file (new function) and returning the `Image` struct from there